### DISCUSSION TITLE
Install Unity64bit on Linux aarch64

### DISCUSSION THREAD

**Discussion by musabafetrat** [ptitSeb/box64#467] [Announcements]
Hello and respect, 
I install [ubuntu aarch64](https://github.com/tuanpham-dev/termux-ubuntu) and [Kali aarch64](https://termux.xyz/how-to-install-kali-linux-on-android-termux-without-root/)  on android with termux.
to install [Unity 64bit](https://unity.com/releases/editor/archive)  inside Ubuntu aarch64 Or Kali aarch64, what exactly should I do?  
I Have Xiaomi 11t.

**Comment by ptitSeb** [comment]
You want to install the full Unity dev. tool?

**Reply by musabafetrat** [reply]
I don't want to output from Unity.  I just want to run and check my ideas in Unity

**Reply by ptitSeb** [reply]
Well, try to install the linux version.
I never tried that, but it might work. It's quite heavy, so it might take a while to load or crash while doing stuff.
If it crashes, open a ticket with log and error message.

**Comment by ptitSeb** [comment]
(I converted the ticket to a discussion)
